### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about baseline policies

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/910

**Comment by msbemba** [comment]
reference # https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/whats-new#replacement-of-baseline-policies-with-security-defaults

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -80,7 +80,10 @@ After you set up multi-factor authentication for your organization, your users w
 
     > [!NOTE]
     > To exclude users from a policy, select **specific users excluded** > **Select excluded users**, select the users from the list, and then choose **Select**.
-
+    
+    > [!NOTE]
+    > As part of a secure-by-default model for authentication, we’re removing the existing baseline protection policies from all tenants. This removal is targeted for completion at the end of February. The replacement for these baseline protection policies is security defaults. If you’ve been using baseline protection policies, you must plan to move to the new security defaults policy or to Conditional Access. If you haven’t used these policies, there is no action for you to take.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
    > As part of a secure-by-default model for authentication, we’re removing the existing baseline protection policies from all tenants. This removal is targeted for completion at the end of February, 2020. The replacement for these baseline protection policies is security defaults. If you’ve been using baseline protection policies, you must plan to move to the new security defaults policy or to Conditional Access. If you haven’t used these policies, there is no action for you to take.
```

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #910.

Thank you.

cc: @kenwith

**Comment by Sirkkuw** [comment]
I am going to close this pull request. the topic in question already has this info and has gone through an edit cycle to update it to the Security defaults.

**Pull request closed without merging**